pokemonterminal/terminal/adapters/terminology.py

```diff
@@ -1,17 +1,18 @@
-import os
+from os import environ
+from subprocess import run
 
 from . import TerminalProvider as _TProv
 
 
 class TerminologyProvider(_TProv):
     def is_compatible() -> bool:
-        return os.environ.get("TERMINOLOGY") == '1'
+        return environ.get("TERMINOLOGY") == '1'
 
     def change_terminal(path: str):
-        os.system('tybg "{}"'.format(path))
+        run(f'tybg "{path}"', check=True)
```

Comment:
This doesn't works in my system. subprocess doesn't accepts strings like this as `os.system` you have to provide then docker style `['exectuable', 'arg1', 'arg2']` otherwise he just tries to resolve the string as an absolute path to an executable.